app/templates/admin/vehicles/upsert.html

```diff
@@ -0,0 +1,142 @@
+{% extends 'admin/base.html' %}
+{% set page_title='Vehicles' %}
+
+{% block action_buttons %}
+  <a href="/admin/vehicles" class="btn btn-link">Back to list</a>
+{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col-8">
+      <form id="vehicleForm" action="/admin/vehicles/save" method="POST" enctype="multipart/form-data">
+        {% if vehicle %}
+          <input name="vehicle_id" type="hidden" value="{{ vehicle['id'] }}"/>
+        {% endif %}
+
+        <div class="mb-3">
+          <label class="form-label">Title</label>
+          <input id="title" type="text" class="form-control" name="title" value="{{ vehicle['title'] if vehicle else '' }}">
+        </div>
+
+        <div class="mb-3">
+          <label class="form-label">Make</label>
+          <input id="make" type="text" class="form-control" name="make" value="{{ vehicle['make'] if vehicle else '' }}">
+        </div>
+
+        <div class="mb-3">
+          <label class="form-label">Model</label>
+          <input id="model" type="text" class="form-control" name="model" value="{{ vehicle['model'] if vehicle else '' }}">
+        </div>
+
+        <div class="mb-3">
+          <label class="form-label">Year</label>
+          <input id="year" type="text" class="form-control" name="year" value="{{ vehicle['year'] if vehicle else '' }}">
+        </div>
+
+        <div class="mb-3">
+          <label class="form-label">Trim</label>
+          <input id="trim" type="text" class="form-control" name="trim" value="{{ vehicle['trim'] if vehicle else '' }}">
+        </div>
+
+        <div class="mb-3">
+          <label class="form-label">Engine</label>
+          <input id="engine" type="text" class="form-control" name="engine" value="{{ vehicle['engine'] if vehicle else '' }}">
+        </div>
+
+        <div class="mb-3">
+          <label class="form-label">Condition</label>
+          <input id="condition" type="text" class="form-control" name="condition" value="{{ vehicle['condition'] if vehicle else '' }}">
+        </div>
+
+        <div class="mb-3">
+          <label class="form-label">Drive</label>
+          <select id="drive" name="drive" class="form-control">
+            <option value="awd" {% if vehicle and vehicle["drive"] == 'awd' %} selected {% endif %}>
+              AWD
+            </option>
+            <option value="fwd" {% if vehicle and vehicle["drive"] == 'fwd' %} selected {% endif %}>
```

Comment:
lets also set fwd as the default option since it is the most common

Reply:
This has fixed